README.md

```diff
@@ -200,6 +200,58 @@ pipenv shell
 apt install lintian
 make test
 ```
+## Signing sources
+
+SecureDrop Workstation code spans across the following repositories:
+
+https://github.com/freedomofpress/securedrop-client
+https://github.com/freedomofpress/securedrop-debian-packaging
+https://github.com/freedomofpress/securedrop-proxy
+https://github.com/freedomofpress/securedrop-sdk
+https://github.com/freedomofpress/securedrop-workstation
+https://github.com/freedomofpress/qubes-template-securedrop-workstation
+
+
+### Release
+1. For each release, a tag will be signed in all the above repos.
+
+2. Create a Makefile target in securedrop-debian-packaging repo that contains release tags / commit hashes for each repository used for the release. To verify the tag signature and check out the packaging logic:
+```
+git tag -v <tag>
+git checkout <tag>
+```
+
+3. Metadata (e.g. commit hash for release) should be tracked inside the .deb (e.g.: `/usr/share/packagename/release-info.txt`)
+
+### Signing binaries/packages
+
+#### Debs
+Apt repository Release file will be signed, containing checksum of the debs.
+
+#### Rpms
+The entire RPM must be signed. This process also requires a Fedora machine/VM on which the GPG signing key (either in GPG keyring or in qubes-split-gpg) is setup.
+
+`rpm -Kv` indicates if digests and sigs are OK. Before signature it should not return signature, and `rpm -qi <file>.rpm` will indicate an empty Signature field.
+
+Set up environment:
+`sudo dnf install rpm-build /usr/bin/rpmsign`
```

Comment:
Double-check that command for me? Looks like `/usr/bin/rpmsign` is a path, rather than a package name, so it should either be removed, or changed to `rpmsign`.

Reply:
It was confusing to me at first, but not a typo, at least for me locally in fedora-28:
```
[user@my-new-fedora ~]$ sudo dnf install rpmsign
Fedora 28 - x86_64 - Updates                    2.7 MB/s |  26 MB     00:09    
Qubes OS Repository for VM (updates)            165 kB/s | 180 kB     00:01    
Last metadata expiration check: 0:00:00 ago on Tue 06 Nov 2018 01:54:16 PM EST.
No match for argument: rpmsign
Error: Unable to find a match
[user@my-new-fedora ~]$ sudo dnf install /usr/bin/rpmsign
Last metadata expiration check: 0:00:16 ago on Tue 06 Nov 2018 01:54:16 PM EST.
Dependencies resolved.
================================================================================
 Package                       Arch      Version               Repository  Size
================================================================================
Installing:
 rpm-sign                      x86_64    4.14.2.1-2.fc28       updates     72 k
Upgrading:
 python2-rpm                   x86_64    4.14.2.1-2.fc28       updates    120 k
 python3-rpm                   x86_64    4.14.2.1-2.fc28       updates    120 k
 rpm                           x86_64    4.14.2.1-2.fc28       updates    533 k
 rpm-build-libs                x86_64    4.14.2.1-2.fc28       updates    138 k
 rpm-libs                      x86_64    4.14.2.1-2.fc28       updates    331 k
 rpm-plugin-selinux            x86_64    4.14.2.1-2.fc28       updates     68 k
 rpm-plugin-systemd-inhibit    x86_64    4.14.2.1-2.fc28       updates     69 k
 rpm-sign-libs                 x86_64    4.14.2.1-2.fc28       updates     74 k

Transaction Summary
================================================================================
Install  1 Package
Upgrade  8 Packages

Total download size: 1.5 M
Is this ok [y/N]: 
```

But reading this again made me discover the package is called `rpm-sign`, thanks ! will change